### PR TITLE
Reworded security section about NDEF signature

### PR DESCRIPTION
**Pull request by beaufortfrancois**
***
<a href="https://pr-preview.s3.amazonaws.com/beaufortfrancois/web-nfc/pull/518.html" title="Last updated on Jan 7, 2020, 10:30 AM UTC (e8ba435)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/518/0a518fc...beaufortfrancois:e8ba435.html" title="Last updated on Jan 7, 2020, 10:30 AM UTC (e8ba435)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4080,11 +4080,9 @@ <h3>Parsing content</h3>
...
 <section> <h2 id="security">Security and Privacy</h2>
   <p>
-    NFC technology involves multiple levels of security. Payments done with NFC
-    are considered to be secure at hardware level, but the whole software stack
-    needs to be security hardened.
-    However, NFC data transfers involve additional threats. In order to mitigate
-    these, the NFC Forum introduced [[NDEF-SIGNATURE]].
+    In order to protect the integrity and authenticity of NDEF messages, the NFC
+    Forum introduced [[NDEF-SIGNATURE]]. Signing NDEF records prevents malicious
+    use of NFC tags.
```
Comment (kenchris) [inline index.html]: but it is also the other way around.

This sounds like you harden the tag instead, but it also allows the reader to only accept such hardened tags and thus hardens the reader as well

Comment (beaufortfrancois) [reply index.html]: That's not how I intended to be understood ;)
How would you rephrase it?

Comment (zolkis) [reply index.html]: I would remove "Signing NDEF records prevents malicious use of NFC tags." 
That is inaccurate.

File: index.html [hunk 2]
```diff
@@ -4080,11 +4080,9 @@ <h3>Parsing content</h3>
 <!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
 <section> <h2 id="security">Security and Privacy</h2>
   <p>
-    NFC technology involves multiple levels of security. Payments done with NFC
-    are considered to be secure at hardware level, but the whole software stack
-    needs to be security hardened.
```
Comment (zolkis) [inline index.html:4085]: I would add a separate sentence, like

"As general security measure,  the whole software stack needs to be security hardened".

And place where you think it adds most clarity/value.

File: index.html (hunk 1 above)
Comment (beaufortfrancois) [reply index.html]: I've removed it.

**Review by zolkis — APPROVED** [review]

File: index.html (hunk 2 above)
Comment (beaufortfrancois) [reply index.html:4085]: This seems obvious to me and not specific to Web NFC, don't you think?

Comment (zolkis) [reply index.html:4085]: IIRC security reviewers insisted on stating the obvious :), but it's the editors' call, indeed.

Comment (beaufortfrancois) [reply index.html:4085]: WDYT of https://github.com/w3c/web-nfc/pull/518/commits/ae9e5471a4ef736b1a1311ba19558fbd17e6685b

**Comment by zolkis** [comment]
We already take that data as untrusted.

**Comment by beaufortfrancois** [comment]
I'll let you update the explainer and I'll remove it.

**Pull request merged**